docs/slides/day1.md

```diff
@@ -48,3 +61,158 @@ Tools needed
 * ruby
     * What version do you have? Run **`ruby --version`** in a terminal
 * irb
+
+---
+
+# Turn This
+
+![repl image](day1_images/repl.png)
+
+---
+
+# Into This
+
+![VS Code & Terminal](day1_images/vs-code-and-terminal.png)
+
+---
+
+# Lets Try VS Code & Terminal
+
+On your computer
+
+- Start up Terminal
+- Create a folder called `Ada` if you do not already have it with:
+  - `$ mkdir Ada`
+- Change into that folder with:
+  - `$ cd Ada`
+- Then create a JSL folder
+  - `$ mkdir JSL`
+  - `$ cd JSL`
+- Then create a day1 folder and move into that folder.
+  - `$ mkdir day1`
+  - `$ cd day1`
+- Create a ruby file and open it in VS Code:
+  - `$ touch hello.rb`
+  - `$ code .`
+
+Then lets create and run this file/
+
+![hello.rb](day1_images/hello-rb.png)
+![Running hello.rb](day1_images/hello-running.png)
+
+---
+
+# Ruby Style
+
+## Goal
+
+- To write Ruby code that is easy to understand and maintain.
+- Tools Needed
+  - [Ruby Style guide https://github.com/rubocop-hq/ruby-style-guide](https://github.com/rubocop-hq/ruby-style-guide)
+
+- Mention of Rubocop
+
+---
+
+# These Two Programs Do The Same Thing
+
+What would you do to improve'em?
+
+![left bad style example](day1_images/left-bad-style.png) ![right bad style example](day1_images/right-bad-style.png)
+
+---
+
+# A Good Code Style Sample
+
+![Good code sample](day1_images/good-code-example.png)
```

Comment:
Comment in code image starts with "Asks the suer..." It should be "Asks the user..."